subsys/nfc/Kconfig.nfc

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
```

Comment:
You can add general configuration option that enables the nfc_platform module.
It should be active by default if CONFIG_NRFXLIB_NFC is set

Reply:
Now this file is inluded only when `CONFIG_NRFXLIB_NFC ` is set